cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaMirrorMaker2ClusterTest.java

```diff
@@ -1391,11 +1390,11 @@ public void testImagePullPolicy() {
         KafkaMirrorMaker2Cluster kmm2 = KafkaMirrorMaker2Cluster.fromCrd(Reconciliation.DUMMY_RECONCILIATION, resource, VERSIONS, SHARED_ENV_PROVIDER);
 
         Deployment dep = kmm2.generateDeployment(
-                replicas, null, Collections.EMPTY_MAP, true, ImagePullPolicy.ALWAYS, null, null);
+                replicas, null, emptyMap(), true, ImagePullPolicy.ALWAYS, null, null);
```

Comment:
Same as above, I updated these lines with emptyMap() instead of Map.of() as it's used in many lines already.